src/Entity/ListBuilder/AppListBuilder.php

```diff
@@ -246,12 +258,22 @@ protected function buildWarningRow(AppInterface $app, array &$rows) {
         'colspan' => count($this->buildHeader()),
       ];
 
+      $items = [];
       if ($warnings['revokedCred']) {
-        $row['data']['info']['data'] = $warnings['revokedCred'];
+        $items[] = $warnings['revokedCred'];
       }
       elseif ($warnings['revokedOrPendingCredProduct']) {
-        $row['data']['info']['data'] = $warnings['revokedOrPendingCredProduct'];
+        $items[] = $warnings['revokedOrPendingCredProduct'];
+      }
+
+      if ($warnings['expiredCred']) {
```

Comment:
should this be an `elseif` or `switch`, or does it need ti be separate from the previous checks in $warnings?

Reply:
@kscheirer It has to be separate because we are always showing expiredCred warnings if they exist.

Reply:
thanks 👍